docs/_layouts/releasepage.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js">
+  <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/clipboard@1/dist/clipboard.min.js"></script>
```

Comment:
you will want to include head.html instead of redefining it here. Better yet, using the existing default layout but including the releases collection  will work better. There shouldn’t be a need to create a new layout.

Reply:
Sorry, I have remove it.